sklearn/cluster/k_medoids_.py

```diff
@@ -0,0 +1,315 @@
+# -*- coding: utf-8 -*-
+"""K-medoids clustering"""
+
+# Authors: Timo Erkkilä <timo.erkkila@gmail.com>
+#          Antti Lehmussola <antti.lehmussola@gmail.com>
+#          Kornel Kiełczewski <kornel.k@plusnet.pl>
+# License: BSD 3 clause
+
+import warnings
+
+import numpy as np
+
+from ..base import BaseEstimator, ClusterMixin, TransformerMixin
+from ..metrics.pairwise import PAIRWISE_DISTANCE_FUNCTIONS
+from ..utils import check_array, check_random_state
+from ..utils.validation import check_is_fitted
+
+
+class KMedoids(BaseEstimator, ClusterMixin, TransformerMixin):
+    """
+    k-medoids class.
```

Comment:
Hmmm, maybe I'll just link to the clustering.rst? I see in K-means there is a 

```
    Read more in the :ref:`User Guide <k_means>`.
```

How does that reference work?